examples/breakout/breakout_stdp.py

```diff
@@ -55,30 +53,28 @@
 )
 
 
-# Train agent for 100 episodes.
+def run_pipeline(pipeline, episode_count):
```

Comment:
This should probably be written to take advantage of the `EnvironmentPipeline.train()` method instead, but I don't particularly feel like doing that atm (just want to fix deprecated `Pipeline` imports rather than make the code perfect).